portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+    public Comment() {
+
+    }
+
+    public Comment(int id, String name, String description, Date postDate) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.postDate = postDate;
+    }
```

Comment:
As I remember, if you follow builder pattern then this class shouldn't have constructors accessible outside. It might conflict with gson library. But please at least remove the one with 4 args.

Reply:
I will do that on PR #14! I don't need this anymore there anyways! :)